plugins/rbac/src/hooks/useSelectedMembers.ts

```diff
@@ -60,6 +64,7 @@ export const useSelectedMembers = (
   return {
     selectedMembers: data,
     members: membersArr,
+    roleDescription,
```

Comment:
will it make sense to return role metadata here and avoid the new hook `useRoleMetadata` or refactor role fetching to `useRoleMetadata` and consume it here

Reply:
Done, I used second option.